src/modules/homepage/SpendFrequency.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Stack } from '@mui/material';
+import { ReactComponent as Graphic } from '../../assets/illustrations/graphic.svg';
+import { theme } from 'src/styles/theme';
+import { colors } from 'src/styles/colors';
+import { Title3 } from '../../shared-components/Typography/Title3';
+
+export const SpendFrequency = () => {
+  return (
+    <Stack>
+      <Box
```

Comment:
Normally, this `Box` element and Title3 inside of it should be a TitleOnly shared component in the Bars folder. 
https://www.figma.com/design/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?node-id=642-11&t=bytnno34hFySyrzH-0

Reply:
I added TitleOnly shared component in Bars folder for this Box and Title3.